File: Selenium/Activities/activity5.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class activity5 {
    public static void main(String[] args) throws InterruptedException {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://www.training-support.net/selenium/input-events");
        System.out.println("Title of the page" +driver.getTitle());
        Actions cube = new Actions(driver);

        cube.click().pause(2000).build().perform();
        String Left_click = driver.findElement(By.className("active")).getText();
        System.out.println(Left_click);

        cube.doubleClick().pause(1000).build().perform();
        String double_click = driver.findElement(By.className("active")).getText();
        System.out.println(double_click);

        cube.contextClick().pause(1500).build().perform();
        String front_cube = driver.findElement(By.className("active")).getText();
        System.out.println(front_cube);
        driver.quit();



    }}

File: Selenium/Activities/activity5.py

from selenium import webdriver
from selenium.webdriver import ActionChains
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
service = FirefoxService(GeckoDriverManager().install())



with webdriver.Firefox(service=service) as driver:

    actions = ActionChains(driver)


    driver.get("https://www.training-support.net/selenium/input-events")


    print("Page title is: ", driver.title)


    actions.click().perform()

    active_side = driver.find_element(By.CLASS_NAME, "active")

    print("left click: ", active_side.text)


    actions.double_click().perform()

    active_side = driver.find_element(By.CLASS_NAME, "active")

    print("double click: ", active_side.text)


    actions.context_click().perform()

    active_side = driver.find_element(By.CLASS_NAME, "active")
    print(" right click: ", active_side.text)